src/application/gtkrs.rs

```diff
@@ -88,7 +89,7 @@ impl App for InnerApplication {
     let cancellable: Option<&Cancellable> = None;
     app.register(cancellable)?;
 
-    let (event_loop_proxy_tx, event_loop_proxy_rx) = channel();
+    let (event_loop_proxy_tx, event_loop_proxy_rx) = unbounded::<Message>();
```

Comment:
Could it still use mpsc channel from std?

Reply:
It has to be an async channel otherwise it blocks the main thread.

Reply:
Even we use `try_recv()` on `event_loop_proxy_rx`?

Reply:
If we did that then we'd have to introduce an artificial sleep into the loop after each iteration and using the normal `thread::sleep` won't work since it'll still block the thread. We'd have to use an async sleep in that case.

Reply:
Okay, I saw it. It does block the thread.